docs/multi-manager-setup.md

```diff
@@ -0,0 +1,154 @@
+# High availability in Swarm
+
+In Docker Swarm, the **swarm manager** is responsible for the entire cluster and manages the resources of multiple `docker` hosts at scale. If the swarm manager dies, you must create a new one and deal with an interruption of service.
```

Comment:
Sorry, missed this the first round;
"`docker` hosts" should be "Docker hosts" here (we're talking about Docker in general here, not the command)